packages/angular/test/apps/ng16/src/app/version-test/bind-component-inputs/bind-component-inputs.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { IonicModule } from '@ionic/angular';
+
+@Component({
+  selector: 'app-bind-route',
+  template: `
+    <ion-header>
+      <ion-toolbar>
+        <ion-title>bindToComponentInputs</ion-title>
+    </ion-toolbar>
+    </ion-header>
+    <ion-content class="ion-padding">
+      <div>
+        <h3>Bind Route</h3>
+        <p id="route-params">Route params: id: {{id}}</p>
+        <p id="query-params">Query params: query: {{query}}</p>
```

Comment:
The query params don't get set when I do `http://localhost:4200/version-test/bind-route/123?liam=hello`

Reply:
For that example to work, you would need to add:
```ts
@Input() liam?: string;
```

to the component class & render it in the template. With the current example, the query parameter would be mapped with a URL like: `http://localhost:4200/version-test/bind-route/123?query=hello`